lib/tdf3/README.md

```diff
@@ -0,0 +1,344 @@
+[![CI Status](https://github.com/opentdf/tdf3-js/actions/workflows/build.yaml/badge.svg)](https://github.com/opentdf/tdf3-js/actions/workflows/build.yaml)
```

Comment:
Is this file used by typedoc or something? You can probably just delete it otherwise